proto/cosmos/distribution/v1beta1/distribution.proto

```diff
@@ -81,7 +81,8 @@ message ValidatorOutstandingRewards {
 // for delegations which are withdrawn after a slash has occurred.
 message ValidatorSlashEvent {
   uint64 validator_period = 1 [(gogoproto.moretags) = "yaml:\"validator_period\""];
-  string fraction = 2 [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Dec", (gogoproto.nullable) = false];
+  string fraction         = 2
+      [(gogoproto.customtype) = "github.com/Finschia/finschia-sdk/types.Dec", (gogoproto.nullable) = false];
```

Comment:
I don't think it's appropriate to format files other than the scope of the current module. It pollutes the PR. Need to discard all changes except settlement module.